resources.whatwg.org/build/deploy.sh

```diff
@@ -49,7 +49,7 @@ if [[ "$TRAVIS" == "true" ]]; then # For some reason the above does not work on
     BRANCH=$TRAVIS_BRANCH
 fi
 
-SERVICE_WORKER_SHA=$(curl https://api.github.com/repos/whatwg/resources.whatwg.org/contents/standard-service-worker.js \
+SERVICE_WORKER_SHA=$(curl https://api.github.com/repos/whatwg/whatwg.org/contents/resources.whatwg.org/standard-service-worker.js \
```

Comment:
This'll fix the problem, but `git hash-object resources.whatwg.org/standard-service-worker.js` would give the same result. @domenic, can I use that? Or just shasum? (The git sha also involves the size of the object, so not the same.)

Reply:
Ha, I'm confused, this script isn't run in the same repo :)